roles/node/vars/main.yml

```diff
@@ -8,6 +8,11 @@
 _node_semver_regex: '^.*((?:0|(?:[1-9]\d*))\.(?:0|(?:[1-9]\d*))\.(?:0|(?:[1-9]\d*))(?:-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(?:\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?).*$'
 _node_binary_version_from_url: "{{ (node_binary | regex_search(_node_semver_regex, '\\1'))[0] }}"
 
+# ID of the GPG public key that is used for signing the polkadot binaries
+_node_binary_release_key_id: 9D4B2B6EB8F97156D19669A9FF0812D491B96798
```

Comment:
This should be in defaults.yml, and expected to be overwritten since there will be use cases outside Polkadot's binaries.